src/app/features/animals/components/animals-form/animals-form.component.html

```diff
@@ -0,0 +1,21 @@
+<h1>Add animal</h1>
+<form (ngSubmit)="submitForm()">
+  <div class="mb-3">
+    <label class="form-label">Name</label>
+    <input type="text" name="name" class="form-control" [(ngModel)]="name">
+  </div>
+  <div class="mb-3">
+    <label class="form-label">Type</label>
+    <select name="type" class="form-select" [(ngModel)]="type">
+      <option value="Domestic">Domestic</option>
```

Comment:
Mas tu na pevno nastavene hodnoty. V taktomto pripade je urcite lepsie pouzit enum. Je to datovy typ prave pre taketo veci.